src/patternfly/components/Nav/nav.scss

```diff
@@ -354,389 +235,275 @@
-    margin-right: var(--pf-c-nav__horizontal-list-item--MarginRight);
-  }
-
-  .pf-c-nav__link {
-    padding-top: var(--pf-c-nav__horizontal-list-link--PaddingTop);
-    padding-bottom: var(--pf-c-nav__horizontal-list-link--PaddingBottom);
-    font-weight: var(--pf-c-nav__horizontal-list-link--FontWeight);
-    color: var(--pf-c-nav__horizontal-list-link--Color);
-    white-space: nowrap;
-
-    // States
-    &.pf-m-hover,
-    &:hover {
-      --pf-c-nav__horizontal-list-link--Color: var(--pf-c-nav__horizontal-list-link--hover--Color);
-
-      &::after {
-        background-color: var(--pf-c-nav__horizontal-list-link--hover--after--BackgroundColor);
-      }
-    }
-
-    &.pf-m-focus,
-    &:focus {
-      --pf-c-nav__horizontal-list-link--Color: var(--pf-c-nav__horizontal-list-link--focus--Color);
-
-      &::after {
-        background-color: var(--pf-c-nav__horizontal-list-link--focus--after--BackgroundColor);
-      }
-    }
-
-    &.pf-m-active,
-    &:active {
-      --pf-c-nav__horizontal-list-link--Color: var(--pf-c-nav__horizontal-list-link--active--Color);
-
-      &::after {
-        background-color: var(--pf-c-nav__horizontal-list-link--active--after--BackgroundColor);
-      }
-    }
-
-    &.pf-m-current {
-      --pf-c-nav__horizontal-list-link--Color: var(--pf-c-nav__horizontal-list-link--m-current--Color);
-
-      &::after {
-        background-color: var(--pf-c-nav__horizontal-list-link--m-current--after--BackgroundColor);
-      }
-    }
-
-    // After - border bottom treatment
-    &::after {
-      position: absolute;
-      bottom: 0;
-      left: 0;
-      display: block;
-      width: 100%;
-      height: var(--pf-c-nav__horizontal-list-link--after--Height);
-      content: "";
-    }
+  --pf-c-nav__link--PaddingTop: var(--pf-c-nav__horizontal-list__link--PaddingTop);
+  --pf-c-nav__link--PaddingRight: var(--pf-c-nav__horizontal-list__link--PaddingRight);
+  --pf-c-nav__link--PaddingBottom: var(--pf-c-nav__horizontal-list__link--PaddingBottom);
+  --pf-c-nav__link--PaddingLeft: var(--pf-c-nav__horizontal-list__link--PaddingLeft);
+  --pf-c-nav__link--Color: var(--pf-c-nav__horizontal-list__link--Color);
+  --pf-c-nav__link--hover--Color: var(--pf-c-nav__horizontal-list__link--hover--Color);
+  --pf-c-nav__link--active--Color: var(--pf-c-nav__horizontal-list__link--active--Color);
+  --pf-c-nav__link--focus--Color: var(--pf-c-nav__horizontal-list__link--focus--Color);
+  --pf-c-nav__link--BackgroundColor: var(--pf-c-nav__horizontal-list__link--BackgroundColor);
+  --pf-c-nav__link--hover--BackgroundColor: var(--pf-c-nav__horizontal-list__link--hover--BackgroundColor);
+  --pf-c-nav__link--active--BackgroundColor: var(--pf-c-nav__horizontal-list__link--hover--BackgroundColor);
+  --pf-c-nav__link--focus--BackgroundColor: var(--pf-c-nav__horizontal-list__link--hover--BackgroundColor);
+  --pf-c-nav__link--m-current--BackgroundColor: var(--pf-c-nav__horizontal-list__link--m-current--BackgroundColor);
```

Comment:
`--pf-c-nav__horizontal-list__link--m-current--BackgroundColor` is undefined. Another approach could be to set the following to transparent by default, then only declare them where they're needed, instead of having to unset them all in horizontal and tertiary nav

```
  --pf-c-nav__link--BackgroundColor
  --pf-c-nav__link--hover--BackgroundColor
  --pf-c-nav__link--active--BackgroundColor
  --pf-c-nav__link--focus--BackgroundColor
  --pf-c-nav__link--m-current--BackgroundColor
```